src/System.Net.Http/tests/StressTests/HttpStress/ChecksumHelpers.cs

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Text;
+using Crc32 = Force.Crc32.Crc32Algorithm;
```

Comment:
There's already a CRC32 implementation at src/Common/tests/System/IO/Compression/CRC.cs, no need for an extra package to restore.

Reply:
I would probably need to copy and modify that file to support the `Append` scenario. Given that the stress suite is a standalone project in the corefx, perhaps avoiding a nuget dependency here is not that important?

Reply:
I suggest just using MD5 checksum like we do for the Azure remote server tests. It's good enough for a checksum.

https://github.com/dotnet/corefx/blob/41cd99d051102be4ed83f4f9105ae9e73aa48b7c/src/System.Net.Http/tests/FunctionalTests/TestHelper.cs#L82-L93

Reply:
@davidsh my only concern with using a crypto hash is that it might overburden the stress run with cpu cycles, to the point where outcomes are altered.